.travis.yml

```diff
@@ -8,15 +8,25 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-25.0.0
-    - android-25
+    - build-tools-28.0.0
+    - android-28
 
     - extra-google-google_play_services
     - extra-google-m2repository
     - extra-android-m2repository
 
   licenses:
+    - 'android-sdk-preview-license-52d11cd2'
     - 'android-sdk-license-.+'
+    - 'google-gdk-license-.+'
+
+before_install:
```

Comment:
I am not entirely sure why we need all these additional configurations, can you simply just try the following
```
  - yes | sdkmanager "platforms;android-28"
```